readme.txt

```diff
@@ -3,10 +3,10 @@ Contributors: maksimer, ledyer
 Tags: woocommerce, ledyer, ecommerce, e-commerce, checkout
 Donate link: https://ledyer.com
 Requires at least: 4.0
-Tested up to: 6.2
+Tested up to: 6.3
 Requires PHP: 7.0
 WC requires at least: 4.0.0
-WC tested up to: 6.9.3
-Stable tag: 0.0.0-development
+WC tested up to: 8.2.0
+Stable tag: 1.7.3
```

Comment:
these should not be changed? Only change locally when deploying for the time being